blockchain/stake/internal/tickettreap/immutable.go

```diff
@@ -278,7 +305,7 @@ func (t *Immutable) Delete(key Key) *Immutable {
 		} else if delNode.right == nil {
 			child = delNode.left
 			isLeft = true
-		} else if delNode.left.priority >= delNode.right.priority {
+		} else if delNode.left.priority <= delNode.right.priority {
```

Comment:
I just wanted to add a comment for the reviewer(s) here - this is a buglet where Put() is trying to maintain it as a min-heap, and Delete() has been trying to maintain it as a max-heap.  

As the heap order is in fact irrelevant in general usage as a treap (as it's a random number), it doesn't actually cause any issues.  Because I did co-opt the priority however, it was an emergent defect when I did that, hence the requirement to make this change.  

It does appear this inconsistency is still in other copies of the treap code in dcrd and btcd.